pkg/mdformatter/linktransformer/link.go

```diff
@@ -286,6 +360,9 @@ func (v *validator) visit(filepath string, dest string, lineNumbers string) {
 	v.destFutures[k] = &futureResult{cases: 1, resultFn: func() error { return nil }}
 	matches := remoteLinkPrefixRe.FindAllStringIndex(dest, 1)
 	if matches == nil {
+		if v.l != nil {
```

Comment:
What about doing it always and just never register/print metrics? (it will be much less distracting)

Reply:
So no `--metrics` flag and it would always be enabled then?

Reply:
Now it does this always but doesn't register if flag not provided! 🙂